dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -2168,7 +2182,7 @@ internal class RumViewScopeTest {
                 .hasAttributes(attributes)
                 .hasUserExtraAttributes(fakeUserInfo.additionalProperties)
                 .hasErrorData {
-                    hasTimestamp(fakeEvent.eventTime.timestamp)
+                    hasTimestamp(fakeEvent.eventTime.timestamp + fakeServerOffset)
```

Comment:
some lines in this file should be replaced by `resolveExpectedTimestamp` for consistency

Reply:
Oh yes...I missed that one...thing is that some are based on a local event declared in the method and we cannot use the utility function ...but maybe I will modify that one to take an event as parameter.